google-oauth-client/src/main/java/com/google/api/client/auth/openidconnect/IdTokenVerifier.java

```diff
@@ -230,14 +227,52 @@ public final Collection<String> getAudience() {
    *       variable set to true.
    * </ul>
    *
-   * <p>Overriding is allowed, but it must call the super implementation.
+   * Deprecated, because returns a false-negatives in case of an error while getting public keys for
```

Comment:
Nit: I would say "This method returns false if network requests to get certificates fail.  Use verifyOrThrow instead to differentiate between potentially retryable network errors and false verification results."

Reply:
Applied

Reply:
Thanks!